src/Apps/W1/Shopify/Test/Companies/ShpfyCompanyMappingTest.Codeunit.al

```diff
@@ -478,11 +483,12 @@ codeunit 134245 "Shpfy Company Mapping Test"
 
     local procedure CreateShopifyCustomer(var ShopifyCustomer: Record "Shpfy Customer"; Id: BigInteger)
```

Comment:
The 1 place where this method is called with id <> 0, you get the random int, same as if it was 0. Basically, you have 1 place where you need to know the id, so just return it. If you don't need it, ignore the return value.

instead of doing an if get then delete, you could set a filter and delete. You don't want the record, just want to make sure the id is not used.

Reply:
don't even need to return it, just get it from the record...